src/components/PageContainer/PageContainer.tsx

```diff
@@ -22,7 +22,7 @@ export const PageContainer = ({ children }: PageContainerProps) => {
         <div className={classes.closeButtonContainer}>
           <button
             className={classes.closeButton}
-            aria-label={String(t('common.close'))}
+            aria-label={String(t('common.close'))} // Should this rather be "Avslutt" or "Cancel" as you are not closing anything?
```

Comment:
Yeah, I see your point here.. 🤔
I think this is inherited from the mentality in A2 where this page was actually a modal. These x-es are usually more associated with closing widows than anything else and I guess you still _could_ argue that you're closing the process you're in, but yeah.. 
I don't actually have a very strong opinion on the matter personally. "Cancel" is probably just as understandable, and probably technically more correct 😊